sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -234,7 +241,9 @@ static const struct dmi_system_id sof_sdw_rt711_rt1308_rt715_quirk_table[] = {
 			DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc"),
 			DMI_MATCH(DMI_PRODUCT_NAME, "XPS"),
 		},
-		.driver_data = (void *)(SOF_RT711_JD_SRC_JD2),
+		.driver_data = (void *)(SOF_RT711_JD_SRC_JD2 |
+					SOF_RT715_DAI_ID_FIX |
```

Comment:
why did you change this? the ID_FIX has nothing to do with this?

Reply:
in original case , one 1308 is in dai 2, another 1308 is in dai 3, and dmic in dai 4. With mult-cpu dai,  two 1308s are in dai 2, so we need to fix dmic in dai 4.